tests/test.sh

```diff
@@ -1,15 +1,32 @@
 #!/bin/bash
 
-if [[ debug == "$1" ]]; then
```

Comment:
You dropped the option to debug ... we need to get hit back ... use some mechanizm to parse CLI variables in bash ...  Look Here "https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash"